burstphoto/align.metal

```diff
@@ -341,9 +658,11 @@ kernel void average_y(texture2d<float, access::read> in_texture [[texture(0)]],
     uint x = gid;
     int texture_height = in_texture.get_height();
```

Comment:
Why does this have to be calculated inside the kernel? Can't this be passed as a parameter?

Reply:
I think Metal is extremely efficient in extracting this information from the texture. Still it can be checked in a future version if performance can be increased.

Reply:
This suggestion does not show any improvement of performance.